src/header/Header.scss

```diff
@@ -1,5 +1,56 @@
+@import "../utils/Mixins";
 @import "../utils/Variables";
 
+@include media-breakpoint-down(sm) {
+  header {
+    flex-wrap: wrap;
+
+    & nav,
+    nav ul,
+    nav ul li {
+      width: 100%;
+    }
+
+    & nav ul {
+      margin-top: 24px;
+    }
+
+    & .navLink {
+      width: 100%;
+    }
+
+    & .networkIndicator {
+      max-width: 68px;
+      font-size: 14px;
+      height: 32px;
+    }
+  }
+
+  .sectionNavigation {
+    order: 3;
+    width: 100%;
+  }
+
+  .sectionStatus {
+    flex: 1 1 auto;
+    justify-content: flex-end;
+
+    & .account {
```

Comment:
I think it doesn't have to be inside '.sectionStatus', it will also work correct